apps/src/templates/instructions/teacherFeedback/TeacherFeedback.jsx

```diff
@@ -255,6 +247,9 @@ export class TeacherFeedback extends Component {
 
     const displayComment = !!comment || viewAs === ViewType.Teacher;
 
+    const displayCommentSection =
```

Comment:
Maybe put this above displayComment?  A comment describing why/how these two values are derived might be helpful too.